contracts/test/strategies/meta.fork-test.js

```diff
@@ -0,0 +1,283 @@
+const { expect } = require("chai");
+
+const { loadFixture } = require("ethereum-waffle");
+const { isForkTest, units, ousdUnits } = require("../helpers");
+const {
+  withBalancedMetaPool,
+  withCRV3TitledMetapool,
+  withOUSDTitledMetapool,
+} = require("../_metastrategies-fixtures");
+
+// Ugly hack to avoid running these tests when running `npx hardhat test` directly.
+// A right way would be to add suffix to files and use patterns to filter
+const forkDescribe = isForkTest ? describe : describe.skip;
```

Comment:
nit: maybe rename this to `forkOnlyDescribe` and you could move it to helpers since it is used in a couple of files